Makefile

```diff
@@ -1,18 +1,20 @@
-.PHONY: test-unit test-e2e
+.PHONY: test-unit test-e2e pytest
 
 test-unit:
 	@echo "Installing test deps..."
 	pip install -r requirements.txt
 	pip install -r requirements-test.txt
 	@echo "Running unit tests..."
-	pytest test_*.py
+	# Command to run unit tests goes here
+	python -m pytest tests/ --junit-xml=tests/test_results/results.xml
 
 test-e2e:
 	@echo "Installing test deps..."
 	pip install -r requirements.txt
 	pip install -r requirements-test.txt
 	@echo "Running end-to-end tests..."
 	# Command to run e2e tests goes here
+	python -m pytest tests/ --junit-xml=tests/test_results/results.xml
```

Comment:
doesn't make sense to run the same set of tests on both targets, let's leave this out for now.

Reply:
we're only even running `test-unit` right now, so I'm not sure we even need the `test-e2e` target -- would it make more sense to zero-out the `test-e2e` target or remove it entirely?

Reply:
zero it out but leave the target.  it reflects our aspirations/intent :)